modules/stats/stats.py

```diff
@@ -252,14 +252,24 @@ def calculate_max_seen_drawdown(self) -> dict:
             "peak": 0,
             "bottom": 0
         }
-        timestamp_value = self.trading_module.open_order_value_per_timestamp
+        timestamp_value_low = self.trading_module.open_trades_low_per_timestamp
+        timestamp_value_open = self.trading_module.open_trades_open_per_timestamp
         timestamp_budget = self.trading_module.budget_per_timestamp
-        for tick in timestamp_budget:
+
+        open_trade_ticks = timestamp_value_low.keys()
+        open_trade_tick_list = list(open_trade_ticks) if open_trade_ticks is not None else []
+        budget_ticks = timestamp_budget.keys()
+        budget_tick_list = list(budget_ticks) if budget_ticks is not None else []
+        ticks = open_trade_tick_list + budget_tick_list
+        ticks.sort()
+
+        for tick in ticks:
             # Find total value at tick time
-            total_value = get_total_value_at_tick(timestamp_value, timestamp_budget, tick)
+            total_value_low = get_total_value_at_tick(timestamp_value_low, timestamp_budget, tick)
+            total_value_open = get_total_value_at_tick(timestamp_value_open, timestamp_budget, tick)
 
             # Check for new drawdown period
-            if total_value > temp_seen_drawdown['peak']:
+            if total_value_open > temp_seen_drawdown['peak']:
                 # If last drawdown was larger than max drawdown, update max drawdown
                 if temp_seen_drawdown['drawdown'] < max_seen_drawdown['drawdown']:
                     max_seen_drawdown['drawdown'] = temp_seen_drawdown['drawdown']
```

Comment:
```suggestion
                    max_seen_drawdown = temp_seen_drawdown
```
Not 100% sure this works but believe it does (and all lines beneath can be removed)